src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,83 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OneToMany
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User user;
+
+    public Order() {
+    }
+
+    public Order(ArrayList<Ticket> tickets, User user, LocalDateTime orderDate) {
+        this.tickets = tickets;
+        this.user = user;
+        this.orderDate = orderDate;
+    }
+
+    public Order(List<Ticket> tickets, User user) {
+        this.tickets = tickets;
+        this.user = user;
+    }
```

Comment:
here you can add something like this:
```suggestion
    public Order(List<Ticket> tickets, User user) {
        this.tickets = tickets;
        this.user = user;
        this.orderDate = LocalDateTime.now();
    }
```